internal/token/jwt.go

```diff
@@ -0,0 +1,78 @@
+package token
+
+import (
+	"errors"
+	"github.com/benbjohnson/clock"
+	"github.com/golang-jwt/jwt/v5"
+	"kg/procurement/cmd/config"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type jwtManager struct {
+	cfg   config.Token
+	clock clock.Clock
+}
+
+func (s *jwtManager) GenerateToken(spec ClaimSpec) (string, error) {
+	if s.cfg.Secret == "" {
+		return "", errors.New("secret key is empty")
+	}
+
+	tokenObject := jwt.NewWithClaims(jwt.SigningMethodHS256, Claims{
+		RegisteredClaims: jwt.RegisteredClaims{
+			Subject:   spec.UserID,
+			ExpiresAt: jwt.NewNumericDate(s.clock.Now().UTC().Add(30 * 24 * time.Hour)), // one month from now
+			IssuedAt:  jwt.NewNumericDate(s.clock.Now().UTC()),
+			ID:        uuid.NewString(),
```

Comment:
Is this ID related to Account ID or no? If it is, we can't use uuid since the ID in Account db table only support varchar(15).

Reply:
No. This is ID for JWT